editors/sc-ide/widgets/code_editor/editor.cpp

```diff
@@ -854,13 +873,23 @@ void GenericCodeEditor::resetFontSize() { mDoc->resetDefaultFont(); }
 
 void GenericCodeEditor::zoomFont(int steps) {
     QFont currentFont = mDoc->defaultFont();
-    const int newSize = currentFont.pointSize() + steps;
-    if (newSize <= 0)
-        return;
-    currentFont.setPointSize(newSize);
+    const int newSize = clampFontSize(currentFont.pointSize() + steps);
+    currentFont.setPointSizeF(newSize);
```

Comment:
Does it matter whether we use `setPoinstSizeF` instead of `setPointsSize` here?

Reply:
Quoting https://doc.qt.io/qt-6/qfont.html#setPointSizeF

> Sets the point size to pointSize. The point size must be greater than zero. The requested precision may not be achieved on all platforms.

IMO this seems preferable in our case

Reply:
I guess my point is that `newSize` is `int` - so why do we use a float function?

Reply:
ah thx, missed to look one line above... changed this to be float throughout